week04/index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="style.css">
+    
+    <title>가위바위보</title>
+</head>
+<body>
+    <div class="top_end"></div>
+    <div class="background">
+        <div class="top">
+            <div class="score">
+                <div id="humanscore">0</div>
+                <div id="vs"> : </div>
+                <div id="computerscore">0</div>
+            </div>
+            <div class="text">버튼을 눌러 시작하세요!</div>
+        </div>
+        
+        <div class="middle">
+            <div id="human_select"><img id="img" src="./img/paper.png"/></div>
+            <div id="robot_select"><img id="robotimg" src="./img/paper.png"/></div>
+        </div>
+        <div class="bottom">
+            <button onclick="scissor_button()" id="img_toScissor">가위</button>
+            <button onclick="rock_button()" id="img_toRrock">바위</button>
+            <button onclick="paper_button()" id="img_toPaper">보</button>
+        </div> 
+        <button id="reset">초기화</button>
+    </div>
+
+    <script>
+        let humanScore  = document.getElementById("humanscore");
+        let computerScore = document.getElementById("computerscore");
+        const resetbutton = document.getElementById("reset");
+        const text = document.getElementsByClassName("text");
+        let human = 0;
+        let computer = 0;
+        resetbutton.onclick = ()=>{
```

Comment:
onclick을 직접적으로 사용하여 함수를 추가하는 것은 제한적이라, 일반적으로 EventListener를 사용합니다.
```suggestion
        function resetScore() {
```